### ISSUE TITLE
examples/neon_cnn in ver. 17.06 failed to run correctly

### ISSUE THREAD

**Issue by cyberfire** [ARM-software/ComputeLibrary#157]
The error message is:
```
!!!!!!!!!!!!!!!!!!!!!!!!!!!

ERROR in configure src/runtime/NEON/functions/NEConvolutionLayer.cpp:137: Output shape does not match the expected one 
!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

I did a little debug, and found this is from the first configuration of conv0.

```cpp
    // in:32x32x1: 5x5 convolution, 8 output features maps (OFM)
    conv0.configure(&src, &weights0, &biases0, &out_conv0, PadStrideInfo());
```

The shape of out_conv0 is still 32x32, while the correct shape should be 28x28.

Thanks,

Cyberfire

**Comment by GeorgeARM** [comment]
Hello @cyberfire,

Yes it seems the shapes in the example are not correct. We will address the issue asap and include the fix in the next maintenance release.

**Comment by OverDriveMC** [comment]
I change the shape of out_conv0 to 28 * 28, It still signals an error, 
what should I do to run the demo correctly?

**Comment by GeorgeARM** [comment]
Should be fixed in release v17.09! Closing. Reopen if needed
